test/fixtures/bootstrappolicy/bootstrap_cluster_roles.yaml

```diff
@@ -1209,6 +1209,13 @@ items:
     verbs:
     - create
     - delete
+  - apiGroups:
+    - ""
```

Comment:
I don't completely understand what this change accomplished, I suppose. But I trust you @deads2k ! :D